testinfra/main.py

```diff
@@ -52,11 +52,12 @@ def report(self):
             status = "OK"
             ret = 0
 
-        sys.stdout.write((
-            b"TESTINFRA %s - %d passed, %d failed, %d skipped in %.2f "
-            b"seconds\n") % (
-                status, self.passed, self.failed, self.skipped, self.total_time
-            ))
+        sys.stdout.write(
+            (
+                b"TESTINFRA %s - %d passed, %d failed, %d skipped in %.2f "
+                b"seconds\n"
+            ) % (status, self.passed, self.failed, self.skipped, self.total_time)
+        )
```

Comment:
Sorry, my commit history got whacked when I was trying to deal with all the merge conflicts when I got back from vacation.  The context here is that the new additions above actually pass flake8 and pylint tests. They wouldn't until I made those changes.